worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/listener/WorldGuardPlayerListener.java

```diff
@@ -168,13 +167,14 @@ public void onPlayerChat(AsyncPlayerChatEvent event) {
     }
 
     @EventHandler(ignoreCancelled = true)
-    public void onPlayerLogin(PlayerLoginEvent event) {
-        Player player = event.getPlayer();
+    public void onPlayerLogin(AsyncPlayerPreLoginEvent event) {
```

Comment:
Does this retain Spigot compatibility? From memory, that was the early concern with moving to this event